client/pages/artists.jsx

```diff
@@ -5,29 +5,47 @@ export default class Artists extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-      displayName: null,
-      timeRange: null,
-      clickedOn: null
+      timeRange: null
     };
     this.handleClick = this.handleClick.bind(this);
   }
 
   handleClick(event) {
-    this.setState({
-      displayName: null,
-      timeRange: null,
-      clickedOn: true
-    });
+    if (event.target.id === 'six-months') {
+      const allTimeFilter = document.querySelector('#all-time');
+      const sixMonthsFilter = document.querySelector('#six-months');
```

Comment:
When building apps with React, we don't query the DOM and update classes, instead have a state property in your component that keeps track of what you want to change about the component, then pass props in your render() method.